src/shared/edit-util.h

```diff
@@ -5,27 +5,42 @@
 
 typedef struct EditFile {
         char *path;
-        char *tmp;
+        char *original_path;
+        char **comment_paths;
+        char *temp;
         unsigned line;
 } EditFile;
 
+typedef struct EditFileContext {
+        EditFile *files;
+        size_t n_files;
+        const char *marker_start;
+        const char *marker_end;
+} EditFileContext;
+
 #define _EDIT_FILE_FOREACH(s, e, i)                                     \
         for (typeof(*(e)) *s, *i = (e); (s = i) && i->path; i++)
```

Comment:
Maybe better to rewrite with EditFileContext.n_files ??

Reply:
I don't know exactly how since one can't initialize variables with different types in a for loop?

Reply:
> CIs are red...

Because of this...